libraries/GROOT/GRootObjectManager.cxx

```diff
@@ -267,12 +267,12 @@ TH1 *GRootObjectManager::GetLast1D(TObject *object) {
 
 
 
-TH2    *GRootObjectManager::GetNext2D(TObject *object) { }
-TH2    *GRootObjectManager::GetLast2D(TObject *object) { }
+TH2    *GRootObjectManager::GetNext2D(TObject *object) { return 0; }
+TH2    *GRootObjectManager::GetLast2D(TObject *object) { return 0; }
```

Comment:
yeah, sorry I am still trying to fill in some of the gaps... at least it is
getting closer


On Fri, May 1, 2015 at 9:20 AM, Ryan Dunlop <notifications@github.com>
wrote:

> In libraries/GROOT/GRootObjectManager.cxx
> <https://github.com/pcbend/GRSISort/pull/298#discussion_r29500826>:
>
> > @@ -267,12 +267,12 @@ TH1 *GRootObjectManager::GetLast1D(TObject *object) {
> >
> >
> >
> > -TH2    *GRootObjectManager::GetNext2D(TObject *object) { }
> > -TH2    *GRootObjectManager::GetLast2D(TObject *object) { }
> > +TH2    *GRootObjectManager::GetNext2D(TObject *object) { return 0; }
> > +TH2    *GRootObjectManager::GetLast2D(TObject *object) { return 0; }
>
> I understand these are just place holders, but may as well make them
> return something for now..Otherwise I get ugly warnings....Mac compiling....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pcbend/GRSISort/pull/298/files#r29500826>.
>